package.json

```diff
@@ -2,12 +2,13 @@
   "name": "libxmljs",
   "author": "Marco Rogers",
   "contributors": [
-    "Jeff Smick"
+    "Jeff Smick",
+    "Martin von Gagern <Martin.vGagern@gmx.net>"
```

Comment:
Sorry, maybe you misunderstood me in a couple of places. Let me try to clarify.

- I don't have any strong opinions, I was just sharing my two cents.
- You said you "don't feel that updating it is required". But, it's not required to *not update* it, right?
- I didn't notice your name wasn't on it. If that's the sticking point, why don't you add it? It would be really easy to do, of course, and the metadata might be useful to others -- you never know. Adding your name, or any other, doesn't lock you in to having to "maintain" it, whatever that means.
- "... determine rules ..." -- why do you need rules? It seems to me it can be as informal as anything else in OSS-land.
- "... maybe you are contributing to a project for the wrong reasons."  Again, all due respect (and I do have a lot of respect for you guys, this is an awesome package) I just think this is a little bit of a backwards way to look at it. I'd prefer to frame it as, "No big deal either way, go ahead and add it." It shouldn't be sticking point to your accepting valuable contributions.

Hope this doesn't upset you -- again, just my two cents. It's your choice, of course!